sdk/tables/data-tables/src/utils/continuationToken.ts

```diff
@@ -5,26 +5,26 @@ import { base64Decode, base64Encode } from "./bufferSerializer";
 
 export interface ContinuationToken {
   nextPartitionKey: string;
-  nextRowKey: string;
+  nextRowKey?: string;
 }
 
 /**
  * Encodes the nextPartitionKey and nextRowKey into a single continuation token
  */
 export function encodeContinuationToken(
-  nextPartitionKey: string = "",
-  nextRowKey: string = ""
+  nextPartitionKey?: string,
+  nextRowKey?: string
 ): string | undefined {
-  if (!nextPartitionKey && !nextRowKey) {
```

Comment:
I would hazard that this is the only line that actually changes runtime behavior, but I could be wrong

Reply:
The issue was caused by the default values we were assigning to nextPartitionKey and nextRowKey in L15 and L16. I'm also removing the default values so L24 would not produce anything is nextRowKey is undefined, before it would add `nextRowKey: ""`

Reply:
Aha! Yes, the default parameter change is important since `""` and `undefined` have different meanings here. Though is `""` ever a valid `nextRowKey`? Should we assert that it is not empty before using it?

Reply:
That's a good point, empty string is not a valid value and the service would give 400. I'll check for truthyness before trying to add it